src/lib/components/SwishHeader.svelte

```diff
@@ -9,11 +11,11 @@
 
 {#if visible}
     <div
-        transition:slide={{ duration: 1000 }}
-        class="relative mb-6 flex h-[166px] w-full rounded-b-3xl {bgcolor} ps-6 pt-12"
+        transition:slide={{ duration: 500, easing: cubicInOut }}
+        class="relative mb-3 flex h-40 w-full flex-shrink-0 rounded-b-3xl {bgcolor} ps-6 pt-12"
     >
-        <div class="absolute bottom-0 left-0 w-full">
-            <img src="/menu-swish.png" alt="Menu Swish" class="h-auto w-full rounded-b-3xl" />
+        <div class="absolute bottom-0 left-0 right-0 top-0">
+            <img src="/menu-swish.png" alt="Menu Swish" class="h-full w-full rounded-b-3xl object-fill" />
```

Comment:
Tweaked some stuff to make the "swish" feel snappier on mobile and work on desktop.